chemprop/features/featurization.py

```diff
@@ -281,6 +299,11 @@ class MolGraph:
     * :code:`b2revb`: A mapping from a bond index to the index of the reverse bond.
     * :code:`overwrite_default_atom_features`: A boolean to overwrite default atom descriptors.
     * :code:`overwrite_default_bond_features`: A boolean to overwrite default bond descriptors.
+    * :code:`is_mol`: A boolean whether the input is a molecule.
+    * :code:`is_reaction`: A boolean whether the molecule is a reaction.
+    * :code:`is_explicit_h`: A boolean whether to retain explicit Hs (for reaction mode)
+    * :code:`is_explicit_h`: A boolean whether to add explicit Hs (not for reaction mode)
```

Comment:
The `is_explicit_h` in line 305 should be `is_adding_hs`?

Reply:
Thank you for catching it! I made a fixup commit [51d0e81](https://github.com/chemprop/chemprop/pull/246/commits/51d0e8132eae154c15bc05d67023b47dd0e4cd80) to correct this typo.